src/main/resources/application.yml

```diff
@@ -139,6 +139,10 @@ dateCalculations:
   waitingTransferDeadlineInDays: 19
   claimantCcjReminderDays: 10
 
+directionDeadline:
+  changeDate: "2020-07-27T11:00:00"
```

Comment:
Can you just once confirm if this should be hardcoded here or taken from environment variable?